sw/device/silicon_creator/manuf/skus/earlgrey_a0/sival_bringup/ft_personalize.c

```diff
@@ -164,6 +170,38 @@ static status_t personalize_otp_secrets(ujson_t *uj) {
   return OK_STATUS();
 }
 
+/**
+ * Sets the attestation binding to the ROM_EXT measurement, and the sealing
+ * binding to all zeros.
+ *
+ * The sealing binding value is set to all zeros as it is unused in the current
+ * personalization flow. This may be changed in the future.
+ */
+static void compute_keymgr_owner_int_binding(manuf_certgen_inputs_t *inputs) {
+  memcpy(attestation_binding_value.data, inputs->rom_ext_measurement,
+         kAttestMeasurementSizeInBytes);
+  memset(sealing_binding_value.data, 0, kAttestMeasurementSizeInBytes);
+}
+
+/**
+ * Sets the attestation binding to a combination of the Owner firmware and
+ * Ownership Manifest measurements, and the sealing binding to all zeros.
+ *
+ * The sealing binding value is set to all zeros as it is unused in the current
+ * personalization flow. This may be changed in the future.
+ */
+static void compute_keymgr_owner_binding(manuf_certgen_inputs_t *inputs) {
+  hmac_digest_t combined_measurements;
+  hmac_sha256_init();
+  hmac_sha256_update((unsigned char *)inputs->owner_measurement,
+                     kAttestMeasurementSizeInBytes);
+  hmac_sha256_update((unsigned char *)inputs->owner_manifest_measurement,
+                     kAttestMeasurementSizeInBytes);
+  memcpy(attestation_binding_value.data, combined_measurements.digest,
+         kAttestMeasurementSizeInBytes);
+  memset(sealing_binding_value.data, 0, kAttestMeasurementSizeInBytes);
+}
+
 /**
  * Crank the keymgr to produce the DICE attestation keys and certificates.
  */
```

Comment:
Should this function also use a `rom_error_t`?

Reply:
Not yet. The manuf functions / provisioning FW still pervasively uses a mix of DIFs, testutils, and silicon_creator code. Until it is moved all to (more efficient) silicon_creator code, we just use the standard `status_t` error type for now.